tests/neg/i10901.scala

```diff
@@ -43,8 +43,8 @@ object BugExp4Point2D {
     val y = DoubleT
 
     val pos1: Point2D[Int,Double] = x º y       // error
-    val pos2: Point2D[Int,Double] = 100 º 200.1 // ok
-    val pos3: Point2D[Int,Double] = 101 º y     // ok
+//  val pos2: Point2D[Int,Double] = 100 º 200.1
+//  val pos3: Point2D[Int,Double] = 101 º y
```

Comment:
could this test be rewritten to enable these tests again, e.g. by manually desugaring the context bounds to how they would be before this PR?